config/options.go

```diff
@@ -666,12 +661,7 @@ func (o *Options) Validate() error {
 	// strip quotes from redirect address (#811)
 	o.HTTPRedirectAddr = strings.Trim(o.HTTPRedirectAddr, `"'`)
 
-	// sort the certificates so we get a consistent hash
-	sort.Slice(o.Certificates, func(i, j int) bool {
```

Comment:
Will we no longer get a consistent hash? Is that a problem?

Reply:
It should be ok because it will only be hashing the strings/filenames and not the actual certificates anymore. Our file watcher should still trigger changes when files change.